### ISSUE TITLE
Codeception tests freezing when using DbSession

### ISSUE THREAD

**Issue by slinstj** [yiisoft/yii2#16429]
### What steps will reproduce the problem?
Configure your session using `DbSession` and run any test using fixtures, for example: `frontend/tests/functional/LoginCest`

### What is the expected result?
Test should run;

### What do you get instead?
Test freezes when is called `codeception\Module\Yii2::_after()`, line 316: `$this->client->getApplication()->session->close();`. 
I think it could be happening because `DbSession` does not implement a `close()` method and it is called from `web\Session` where we have `YII_DEBUG ? session_write_close() : @session_write_close();` and it does the running to freeze. Not sure why, it is just a guessing.

### Additional info

| Q                | A
| ---------------- | ---
| Yii version      | 2.0.15.1
| PHP version      |  7.1.14
| Operating system | Win 10

**Comment by slinstj** [comment]
Tryed the exact same project in my home (environment Linux and MySQL) with no freezing!

So, this can be something specific to Win10 or SQL Server.

**Comment by samdark** [comment]
Then closing the issue. If you'll find out what it is, please re-open. Thanks.

**Comment by slinstj** [comment]
No problem, @samdark.

I tryed again in Win10 but using MySQL this time and tests works like expected. So, I can see two problems till here:

1. I implemented a method `close()` for DbSession just returning true and I had no more freezes in that point... 
2. Now the code freezes later, when running a long sql on `checkIntegrity`. So, we have some problem MSSQL related. I tryed to debug it but the problem happens when calling `pdo->execute()`;

An strange thing that happens is that, if I `echo` anything just before `Command` `pdo->execute()` there is no freeze at all and the tests are finished. However, doing this, I have error about "sent data before set header" (like expected).

Nothing makes any sense about those errors. :-/

I will have to use MySQL to run tests for now.
